apps/deploy-web/src/utils/TransactionMessageData.ts

```diff
@@ -221,7 +221,7 @@ export class TransactionMessageData {
           ],
           expiration: expiration
             ? {
-                seconds: Long.fromInt(Math.floor(expiration.getTime() / 1_000)) as unknown as Long,
+                seconds: BigInt(Math.floor(expiration.getTime() / 1_000)),
```

Comment:
I think it was fine with `Long.fromInt`, any reason for the change?

Reply:
yes, update package with different types

Reply:
Ah ok